docs/process/Development.md

```diff
@@ -17,10 +17,12 @@ You are responsible for shepherding your work through the development process. I
 > Issue transitions to **In Review**
 
 - Notify other devs and wait for reviews
+- Add the **needs test** label
 - _If changes are requested:_
    - make the changes
    - notify the reviewer by tagging them in a comment
-- **Once you have two approvals and one test:**
+- (When a test is successful, the reviewer who tested it should remove the **needs test** label)
```

Comment:
maybe format this somehow better than parentheses? doesn't seem to fit, quite. this may actually better belong in a `Testing` doc, which we don't have yet, so i guess we can move it when we make that doc. what do you think?

Reply:
The testing doc would probably be where this should go.